JavaScript/JavaScriptSDK/Telemetry/RemoteDependencyData.ts

```diff
@@ -39,8 +40,10 @@ module Microsoft.ApplicationInsights.Telemetry {
             super();
 
             this.id = id;
-            this.name = this.formatDependencyName(method, absoluteUrl);
-            this.commandName = Common.DataSanitizer.sanitizeUrl(commandName);
+            this.target = UrlHelper.parseUrl(absoluteUrl).hostname;
+            this.commandName = this.formatDependencyName(method, absoluteUrl);
```

Comment:
I believe the transformation is handled in breeze? I though breeze today expects commandName  as opposed to data?